package/pyproject.toml

```diff
@@ -12,3 +12,63 @@ requires = [
     "setuptools",
     "wheel",
 ]
+
+[project]
+name = "MDAnalysis"
+version = "2.1.0-dev0"
+license = {file = "LICENSE"}
+authors = [
+    {name = 'MDAnalysis Development Team', email = 'mdanalysis@numfocus.org'}
+]
+maintainers = [
+    {name = 'MDAnalysis Core Developers', email = 'mdanalysis@numfocus.org'}
+]
+requires-python = ">=3.7"
+dependencies = [
+    'numpy>=1.18.0',
+    'biopython>=1.71',
+    'networkx>=1.0',
+    'GridDataFormats>=0.4.0',
+    'mmtf-python>=1.0.0',
+    'joblib>=0.12',
+    'scipy>=1.0.0',
+    'matplotlib>=1.5.1',
+    'tqdm>=4.43.0',
+    'threadpoolctl',
+]
+readme = "../README.rst"
+keywords = [
+    "python", "science", "chemistry", "biophysics", "molecular-dynamics",
+    "computational-chemistry", "molecular-simulation", "analysis",
+    "trajectory-analysis",
+]
+classifiers = [
+    'Development Status :: 6 - Mature',
+    'Environment :: Console',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
```

Comment:
@orbeckst I'm correct in thinking we can do this change right?

Reply:
@orbeckst can I ping you on this again please?

Reply:
Sorry, must have overlooked the notifications. 

Yes, that's excellent. I think I wasn't aware that there was a classifier for "GPL v2-or-later".